### DISCUSSION TITLE
WGS84(EPSG:4326) to ETRS84(EPSG:4258) Accuracy

### DISCUSSION THREAD

**Discussion by Tlunch** [pyproj4/pyproj#1303] [Q&A]
I am aiming to transform from the WGS84 datum to the ETRS89 datum with  October 2022 data with sub centimeter accuracy but i seem to be getting back data that is still offset by about a meter. What are the best practices for precision and where might I be going wrong?

My CRSs 
`etrs89_crs = pyproj.CRS.from_authority('epsg', '4258')`
`wsg84_crs = pyproj.CRS.from_authority('epsg', '4326')`

my transformer
`wsg84_to_etrs89 = Transformer.from_crs(wsg84_crs, etrs89_crs, only_best=True)`

the application of the transformer
`out_1, out_2, out_3, year = transformer.transform(first_axis, second_axis, altitude, tt = 2022.0, radians=False)`

**Comment by snowman2** [comment]
With the [accuracy](https://pyproj4.github.io/pyproj/stable/api/transformer.html#pyproj.transformer.Transformer.accuracy) property, you can see the expected accuracy of the transformation:
```python
>>> wsg84_to_etrs89.accuracy
1.0
```
I think this matches up with the 1 meter offset you are seeing.

**Reply by Tlunch** [reply]
Is there any way to specify a more precise transform? Also my transform description is showing a noop -
`<Transformation Transformer: noop>`
`Description: Inverse of ETRS89 to WGS 84 (1)`
does this mean this transform is doing nothing?
If so how do I set it to perform a transform as these datums are almost a meter offset.

**Reply by snowman2** [reply]
You can see what transformations are available in PROJ using:
- [projinfo](https://proj.org/en/stable/apps/projinfo.html)
- [TransformerGroup](https://pyproj4.github.io/pyproj/stable/api/transformer.html#pyproj.transformer.TransformerGroup)

**Reply by Tlunch** [reply]
From my understanding this transform should be available to the module. [url](https://epsg.org/transformation_9225/WGS-84-to-ETRS89-2.html?sessionkey=hmqa81lqit) 
<img width="950" alt="image" src="https://github.com/pyproj4/pyproj/assets/89159592/28084a0e-433c-4cde-a659-0a187acd34b4">
But it seems that the module is using the inverse of this transform. [url](https://epsg.org/transformation_1149/ETRS89-to-WGS-84-1.html?sessionkey=u4v29y29cu)
<img width="950" alt="image" src="https://github.com/pyproj4/pyproj/assets/89159592/ef2cc332-4eeb-4789-97e2-ad3ef9d35dab">
Even though that the second has a 1m accuracy and is a noop and the first has a 0.1 accuracy and is an actual operation. This seems like unintended behavior?

I have fixed the problem for my project by implementing the matrix transformation manually.

**Reply by Tlunch** [reply]
Sorry if the URLs don't work.
